notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -122,7 +122,7 @@
       "source": [
```

Comment:
<h2>managed tabular dataset -&gt; Vertex AI Tabular Dataset</h2><p> </p><p>Add explain of call and params</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>updated...</p>